projects/lib/src/lib/pipes.ts

```diff
@@ -95,6 +95,26 @@ export class IsSuccessPipe implements PipeTransform {
   }
 }
 
+@Pipe({ name: 'hasValue' })
+export class HasValuePipe implements PipeTransform {
```

Comment:
Why do we need this pipe?

Reply:
This would be much more convenient than, lets say:
*ngIf="(myItem | isSuccess) || ((myItem | isInProgress) && (myItem | inProgressValue) !== null && (myItem | inProgressValue) !== undefined)"

Reply:
Ok, it makes sense 👍